vtm.js

```diff
@@ -183,15 +206,30 @@ app.component('stat',{
         var value = this.stat.value;
         var i = this.hoverPointer;
         var diff = i - value;
+        let tmp;
         
-
+        // -1 if not found || i if found [0, index]
```

Comment:
pomohol by komentar vysvetlujuci, co sa teraz ma robit. Chapem, ze sa nieco kontroluje, ale nechapem co a preco

Reply:
opet nechapem tvoj komentar vzhladom ku kodu hore

Reply:
opat je tu nejaky kod, ktory pracuje s poliami a premennymi, u ktoreho na prvy pohlad neni jasne k comu presne sluzi. Prosim o doplnenie nejakej dokumentacie

Reply:
no dobre a ja ti hovorim ze je to pre mna neintuitivne lebo nevidim na co sa pytas len nejaky komentar, ktory asi tusim k comu patri, a ak je to ten kod kery myslim, tak je to nieco na principe bubblesortu a neviem ci to tu mam vysvetlovat

Reply:
mozem ti asi poradit dat tam debugger a pozret sa co to robi

Reply:
predstavoval by som si komentar funkcie handleClick v docstring formate:

/** 
* Brief description of the function here.
* @summary If the description is long, write your summary here. Otherwise, feel free to remove this.
* @param {ParamDataTypeHere} parameterNameHere - Brief description of the parameter here. Note: For other notations of data types, please refer to JSDocs: DataTypes command.
* @return {ReturnValueDataTypeHere} Brief description of the returning value here.
*/

netreba vysvetlovat ako funguje bubblesort, mozet ti poslat vysvetlenie bubblesortu ako madarsky krojovy tanec :-) ale nevidim ako presne sa mapuju cisla v tom poli na ty omezenia a ako sa k tomu vztahuje ta funkcia. 

>  pre mna neintuitivne lebo nevidim na co sa pytas len nejaky komentar, ktory asi tusim k comu patri, a ak je to ten kod kery
njn, suhlasim, ze ja sa furt v druhom okne divam do IDEcka a hladam ten kod tam. Mozem zkusit pohladat nastavenie, aby github zobrazoval viacej riadkov kodu

Reply:
>Mozem zkusit pohladat nastavenie, aby github zobrazoval viacej riadkov kodu

nenasiel

Reply:
... fajn takze to budem vysvetlovat ako bubblesort
chces pridat attribut v cerstvo obnovenej stranke; klik strenght 4;` addcheck([0,-8,4,3,1, 0], 4);` `array[4] > 0 ? true` => `return 4;`pole sa zmeni na [0,-7,4,3,0,0], strength-value sa zmeni na 4;
klik stamina 5; `addcheck(`to pole hore`, 5)`; `array[5] > 0 ?`false => `array[4]>0 ?` false => `array[3] > 0 ?` true => `return 3`
btw to s tym madarskym tancom som ti najskor posielal ja lebo to nam ukazovali na strednej pred asi 7 rokmi ked nas ucili bubblesort

Reply:
ok, fajn, no a nebolo by mozne dorobit ten docstring? Ako som chcel...

Neviem, ci si dobre rozumieme, pisem, ze neni treba vysvetlovat bubblesort a ty napises, ze budes vysvetlovat ako bubblesort...

Reply:
ja neviem co je docstring, nemozes to dopisat ty?

Reply:
ja by som rad, ale ja neviem, ako to funguje - doc string je retazec, ktory popisuje vlastnosti funkcie, priklad viz komentar co som pisal o kusok vyssie - https://github.com/haterzlin/character-sheet/pull/37#discussion_r669324679

Reply:
https://en.wikipedia.org/wiki/Docstring

Reply:
alebo sa pozri sem: https://www.php.net/manual/en/function.in-array.php
je tam napisane co funkcia robi, jednotlive parametre su popisane, co je co a je tam aj napisane, co funkcia vracia